### PR TITLE
feat: upgrade to node 22 lts

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/skyleague/node-standards/213?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#213** <a href="https://app.graphite.dev/github/pr/skyleague/node-standards/213?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/skyleague/node-standards/213?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#212** <a href="https://app.graphite.dev/github/pr/skyleague/node-standards/212?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: .github/workflows/reusable-release.yml [hunk 1]
```diff
@@ -85,7 +85,7 @@ jobs:
         env:
           GITHUB_NPM_TOKEN: ${{ secrets.GITHUB_NPM_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - run: npm run build:docs
+      - run: node --run build:docs
```
Comment (MickVanDuijn) [inline .github/workflows/reusable-release.yml:88]: will this work if we pin the pipeline to an older version of Node?

**Review by MickVanDuijn — APPROVED** [review]

File: .github/workflows/reusable-release.yml (hunk 1 above)
Comment (pevisscher) [reply .github/workflows/reusable-release.yml:88]: No, but do we care? Node 22 is the new LTS, so in my opinion we should just move everything to that anyway

**Comment by pevisscher** [comment]
### Merge activity

* **Dec 3, 1:16 PM GMT+1**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/skyleague/node-standards/213).

**Pull request merged**